draft-packages/form/KaizenDraft/Form/Primitives/InputSearch/InputSearch.tsx

```diff
@@ -0,0 +1,104 @@
+import { Icon } from "@kaizen/component-library"
+import { LoadingSpinner } from "@kaizen/draft-loading-spinner"
+import classnames from "classnames"
+import React, { useState } from "react"
+import search from "@kaizen/component-library/icons/search.icon.svg"
+import clear from "@kaizen/component-library/icons/clear.icon.svg"
+
+import styles from "./styles.scss"
+
+export interface InputSearchProps
+  extends Omit<React.InputHTMLAttributes<HTMLInputElement>, "className"> {
+  classNameAndIHaveSpokenToDST?: string
+  inputRef?: React.RefObject<HTMLInputElement>
+  reversed?: boolean
+  loading?: boolean
+  primary?: boolean
+  secondary?: boolean
+  startIconAdornment?: React.ReactNode
+  endIconAdornment?: React.ReactNode
```

Comment:
Don't think you need these `iconAdornment` props, they're not actually being used. On `Input` they are used to allow the `TextField` (or a direct usage) to pass an icon through to it, but we're not allowing that flexibility on this component.